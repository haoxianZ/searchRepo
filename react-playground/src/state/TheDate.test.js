import React from 'react';
import renderer from 'react-test-renderer';
import TheDate from './TheDate';
import ReactDOM from 'react-dom';

describe('The Date component', () => {
    it('render without crashing', () =>{
        const div =document.createElement('div');
        ReactDOM.render(<TheDate />, div);
        ReactDOM.unmountComponentAtNode(div);
    })
    it.skip('renders the UI as expected', ()=>{
        expect(
            renderer.create(<TheDate />).toJSON())
            .toMatchSnapshot()
    })
})