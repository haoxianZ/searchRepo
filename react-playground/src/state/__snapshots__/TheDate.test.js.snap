// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Date component renders the UI as expected 1`] = `
<div>
  8/4/2020, 10:57:31 AM
</div>
`;
